tests/unit/test_dispatcher.py

```diff
@@ -17,6 +19,7 @@ def return_fake_bot_id():
         return FAKE_BOT_ID
     dispatcher = MessageDispatcher(None, None)
     monkeypatch.setattr(dispatcher, '_get_bot_id', return_fake_bot_id)
+    monkeypatch.setattr(dispatcher, '_get_bot_name', lambda: FAKE_BOT_NAME)
```

Comment:
prob should do the same with return_fake_bot_id, that lambda is indeed more elegant, nice.